### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henrymbuguakiarie**
## Purpose
Adding clarity to the README, we don't have user_impersonation as an API permission.

## Does this introduce a breaking change?

```
[ ] Yes
[N] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by henrymbuguakiarie** [comment]
Hello @ckittel,

Please review proposed change.

Cc: @mmacy, @skabou, @Dickson-mwendia

**Review by skabou — APPROVED** [review]

**Comment by ckittel** [comment]
Typically we present them in this format:

image

https://github.com/Azure-Samples/ms-identity-docs-code-python/tree/main/src/protect-function-app#1-register-the-app

Since we need to list all the fields a user needs to fill in.  Is there a reason we are only showing one of the fields we need?

File: protected-api-access-protected-api/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Use these settings in your app registration.
 | **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
 | **Platform type**                 | _None_                                                                      | No redirect URI required; don't select a platform.                                                                    |
 | **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
-| **API Permissions**               | `user_impersonation`                                                         | Create a new permission called user_impersonation.  Required value for this sample.      |
+| **Scope**               | `user_impersonation`                                                         | Create a new permission called user_impersonation.  Required value for this sample.      |
```
Comment (skabou) [inline protected-api-access-protected-api/README.md:63]:
```suggestion
| Scopes defined by this API               | Scope name: `user_impersonation`
Who can consent?: Admins and users
Admin consent display name: `User Impersonation`
Admin consent description: `Allows the application to perform an action on behalf of the user`
User consent display name: `User Impersonation`
User consent description: `Allows the application to perform an action on behalf of the user`
State: Enabled                                                          | Required scope for this sample.      |
```

Comment (skabou) [inline protected-api-access-protected-api/README.md:63]:
```suggestion
| Scopes defined by this API        | Scope name: `user_impersonation`<br/>Who can consent?: Admins and users<br/>Admin consent display name: `User Impersonation`<br/>Admin consent description: `Allows the application to perform an action on behalf of the user.`<br/>User consent display name: `User Impersonation`<br/>User consent description: `Allows the application to perform an action on behalf of the user.`<br/>State: Enabled                                                          | Required scope for this sample.      |
```

Comment (henrymbuguakiarie) [reply protected-api-access-protected-api/README.md]: @skabou Thanks for suggestion.

**Review by skabou — APPROVED** [review]
Looks good

**Review by ckittel — APPROVED** [review]
:shipit:

**Pull request merged**